src/io/flutter/actions/FlutterExternalIdeActionGroup.java

```diff
@@ -47,6 +47,14 @@ protected static boolean isIOsDirectory(@NotNull VirtualFile file) {
   }
 
   private static boolean isProjectDirectory(@NotNull VirtualFile file, @Nullable Project project) {
-    return file.isDirectory() && project != null && project.getBaseDir().getPath().equals(file.getPath());
+    if (!file.isDirectory()) {
+      return false;
+    }
+    //noinspection SimplifiableIfStatement
+    if (project == null || project.getBaseDir() == null) {
```

Comment:
Probably not a big deal, but perhaps consider doing the same here?

Reply:
good catch; updated